tailwind.config.js

```diff
@@ -1,7 +1,36 @@
 const plugin = require('tailwindcss/plugin')
 
+const borderRadius = Object.fromEntries(
+  Object.values(require('./tokens/dist/js/rounded').rounded)
+    // Filter out aliases here because they are only included for backward compatibility with Scss
+    .filter(({ original}) => original.value.includes('{') === false)
+    .map(({ attributes, value }) => [attributes.type, value])
+)
+
+const boxShadow = Object.fromEntries(
```

Comment:
Nitpick: We could extract this code into a separate function to reduce code duplication

Reply:
right! 5a845e4f8f9157113e7c62e3c2ebad5596413bb6